server/app/modules/DurableJobModule.java

```diff
@@ -89,6 +90,11 @@ public DurableJobRegistry provideDurableJobRegistry(
             new UnusedAccountCleanupJob(accountRepository, nowProvider, persistedDurableJob),
         new RecurringJobExecutionTimeResolvers.SecondOfMonth2Am());
 
+    durableJobRegistry.register(
+        DurableJobName.FIX_APPLICANT_DOB_DATA_PATH,
+        persistedDurableJob -> new FixApplicantDobDataPathJob(persistedDurableJob),
+        new RecurringJobExecutionTimeResolvers.Sunday2Am());
```

Comment:
What do you think about my plan to make this a recurring weekly job and clean it up in a month to ensure it had the chance to run for everyone?

Reply:
SGTM

suggestion (non-blocking): run it nightly instead of once a week to reduce time to incident resolution.

Reply:
done